ksql-engine/src/test/java/io/confluent/ksql/integration/StreamsSelectAndProjectIntTest.java

```diff
@@ -117,35 +214,61 @@ public void testSelectProject() throws Exception {
 
 
   @Test
-  public void testSelectStar() throws Exception {
+  public void testSelectProjectAvroJson() throws Exception {
```

Comment:
should this be `testSelectProjectAvro`

Reply:
This is for testing format change from avro to json.